src/nitro/communication/messages/parser/userclassification/UserClassificationMessageParser.ts

```diff
@@ -0,0 +1,59 @@
+import { IMessageDataWrapper, IMessageParser } from '../../../../../api';
+
+export class UserClassificationMessageParser implements IMessageParser
+{
+    private _classifiedUsersNames: Map<number, string>;
+    private _classifiedUsersClass: Map<number, string>;
+
+    public flush(): boolean
+    {
+        if(this._classifiedUsersNames)
+        {
+            this._classifiedUsersNames = new Map();
+        }
+        if(this._classifiedUsersClass)
+        {
+            this._classifiedUsersClass = new Map();
+        }
+
+        return true;
+    }
+
+    public parse(wrapper: IMessageDataWrapper): boolean
+    {
+        if(!wrapper) return false;
+
+        let _local_3: number;
+        let _local_4: string;
+        let _local_5: string;
+
+        const _local_2: number = wrapper.readInt();
+
+        this._classifiedUsersNames = new Map();
+        this._classifiedUsersClass = new Map();
+
+        let _local_6: number;
```

Comment:
initialize `_local_6` to 0 since ActionScript does this implicitly, but typescript wont